ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -49,7 +53,16 @@ def __init__(self, *args, **kwargs):
         self.metadata_collector = MetadataCollector(self._config, self.log)
         self.stats_collector = StatsCollector(self._config, self.send_metrics_from_properties, self.log)
 
+        self.queue_manager_process_pattern = None
+
+        self.check_initializations.append(self.parse_config)
```

Comment:
Why not do this in the `IBMMQConfig` object?

Reply:
https://datadoghq.dev/integrations-core/base/basics/#check-initializations

Reply:
Parse config could still live inside IBMMQConfig object

Reply:
I still think queue_manager_process_pattern should live inside IBMMQConfig